config/crd/bases/druid.gardener.cloud_etcds.yaml

```diff
@@ -398,7 +406,7 @@ spec:
                   #default: 10485760
                 deltaSnapshotPeriod:
                   type: string
-                  #default: 60s 
+                  default: 60s
```

Comment:
defaults are in alpha and will work from 1.16

Reply:
Thanks for info. I just kind of cherry picked from old PR. Will go though it again. Also, i will try to change the CRD version from v1 to v1alpha1